http/src/main/scala/org/http4s/blaze/http/websocket/WSFrameAggregator.scala

```diff
@@ -15,7 +15,7 @@ class WSFrameAggregator extends MidStage[WebSocketFrame, WebSocketFrame] {
 
   def name: String = "WebSocket Frame Aggregator"
 
-  private val queue = new ListBuffer[WebSocketFrame]
+  private var queue = new ArrayBuffer[WebSocketFrame]
```

Comment:
When you need a mutable reference to a mutable value, you know you're in the shit.